intro-HPC/ch_introduction.tex

```diff
@@ -203,7 +203,7 @@ \section{What is the \hpcInfra?}
 %TODO check tools for other sites
 
 
-\section{What is the \hpc not!}
+\section{What is the HPC infrastucture not?}
```

Comment:
`What the HPC infrastructure is \emph{not}`

`The HPC infrastructure is \emph{not} a magic computer that automatically:`

(not sure if the `magic` bit should be there)